### ISSUE TITLE
Scene closes itself. No main.jar detected

### ISSUE THREAD

**Issue by moraguma** [utopia-rise/godot-kotlin-jvm#280]
**Version:** 0.2.0-3.3.2
**Platform:** Windows

I followed the [getting started guide](https://godot-kotl.in/en/stable/getting-started/requirements/) to make an initial project, but when I try to run the scene it just closes itself and I get the following error:

![image](https://user-images.githubusercontent.com/63124011/139516424-8949456b-5c75-4f3e-bf6b-1fa672e0fce3.png)

I have run gradlew build on the IntelliJ terminal. The scene is a Spatial node with the [tutorial class](https://godot-kotl.in/en/stable/getting-started/your-first-class/) attached. It runs fine without the script attached.

![image](https://user-images.githubusercontent.com/63124011/139516671-e584922a-081b-41cd-8292-fc8f5704d726.png)

**Comment by piiertho** [comment]
Hello !
Can you check `main.jar` is present in `build/libs` folder of your Godot project ? 
Can you also post output of gradle build ?
Also you should create a jre in your project (even if this is not the source of your problem, I think)

**Comment by moraguma** [comment]
Hey, thanks, I figured out what I was doing wrong. I had the IntelliJ workspace and Godot project in different folders when they should have been in the same one, the scene is running as expected now.

I think it would be nice if the getting started guide had some indication of what your folder structure is supposed to look like at the end. I wasn't really sure where I was supposed to setup the Godot project so I just ended up guessing (wrongly).
